### PR TITLE
Add new doamin PetInsurance and update /schema/index.js

### PR DESCRIPTION
**Pull request by Xedros**

### CODE REVIEW

**Comment by inca** [comment]
Circle didn't want to build forked PRs by default, so I changed that. Will close and reopen

File: schema/pet-insurance.json [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+                    "price"
+                ]
+            
+        },
+        "Owner": {
+            "type": "object",
+            "pii": true,
+            "additionalProperties": false,
+            "properties": {
+                "person": {
```
Comment (randunel) [inline schema/pet-insurance.json]: Use `Generic.types.Person`

File: schema/pet-insurance.json [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+                        "dateOfBirth": {
+                            "type": "string",
+                            "description": "Passenger's date of birth in YYYY-MM-DD format.",
+                            "format": "date",
+                            "example": "1976-01-27"
+                        }
+                    }
+                }
+            },
+
```
Comment (randunel) [inline schema/pet-insurance.json]: Why is there an empty line here?

File: schema/pet-insurance.json [hunk 3]
```diff
@@ -0,0 +1,596 @@
...
+                        "number": {
+                            "type": "string",
+                            "description": "",
+                            "example": "123"
+                        },
+                        "name": {
+                            "type": "string",
+                            "default": "",
+                            "description": ""
+
```
Comment (randunel) [inline schema/pet-insurance.json]: Empty line?

File: schema/pet-insurance.json [hunk 4]
```diff
@@ -0,0 +1,596 @@
...
+                            "description": "",
+                            "example": "W1W 5BB"
+                        }
+                    }
+                }
+            }
+
+        },
+
+
```
Comment (randunel) [inline schema/pet-insurance.json]: 2 empty lines? @Xedros please maintain a clean code base.

File: schema/pet-insurance.json [hunk 5]
```diff
@@ -0,0 +1,596 @@
...
+            "description": "The policy has already been purchased."
+        },
+        {
+            "code": "BreedTypeRequired",
+            "category": "client",
+            "description": "Breed type is required by the website"
+        }
+    ]
+
+}
```
Comment (randunel) [inline schema/pet-insurance.json]: All lines must end with newline. Configure your editor.

**Comment by randunel** [comment]
Please add `example` to explicit data types and `description` to inputs and outputs.

**Pull request merged**